docs/eventing/debugging/README.md

```diff
@@ -61,6 +61,349 @@ LAST SEEN   FIRST SEEN   COUNT     NAME                             KIND      SU
 
 ## Where are my events?
 
+You can check if events were delivered by lokking int logs of sunscribers.
+
+```
+kubectl -n knative-debug get pods
+NAME                                             READY   STATUS    RESTARTS   AGE
+apiserversource-src-skzlm-54c6bbdccd-w27ms       1/1     Running   0          19m
+event-display-prxh4-deployment-89c744b59-kv8zt   2/2     Running   0          91s
+fn-bfb44f975-2s68r                               1/1     Running   0          64m
+
+kubectl -n knative-debug -c user-container logs event-display-prxh4-deployment-89c744b59-45d8g|less
+☁️  CloudEvent: valid ✅
+Context Attributes,
```

Comment:
This is a _really_ long event. What if we switch the ApiServerSource to something much simpler than a Pod, e.g. a K8s Service. We can still create and delete it on the command line easily and the generated event will be much shorter.